core/src/main/java/com/ritense/valtimo/security/config/UserHttpSecurityConfigurer.java

```diff
@@ -40,7 +41,9 @@ public void configure(HttpSecurity http) {
                 .antMatchers(GET, "/api/v1/users/{userId}").hasAuthority(ADMIN)
                 .antMatchers(GET, "/api/v1/users/authority/{authority}").hasAuthority(ADMIN)
                 .antMatchers(DELETE, "/api/v1/users/{userId}").hasAuthority(ADMIN)
-                .antMatchers(POST, "/api/v1/users/send-verification-email/{userId}").hasAuthority(ADMIN);
+                .antMatchers(POST, "/api/v1/users/send-verification-email/{userId}").hasAuthority(ADMIN)
+                .antMatchers(GET, "/api/v1/user/settings").hasAuthority(USER)
+                .antMatchers(POST, "/api/v1/user/settings").hasAuthority(USER);
```

Comment:
```suggestion
                .antMatchers(PUT, "/api/v1/user/settings").hasAuthority(USER);
```

This doesn't look up to standards